packages/block-library/src/button/editor.scss

```diff
@@ -91,3 +75,7 @@
 		margin-top: $grid-size-large;
 	}
 }
+
+.wp-block-button-wrapper {
+	display: table;
```

Comment:
Is `display: table;` the only way to solve this? It seems like a bit of a hack that'll make life a lot harder in the long run.